webapp/src/components/common/DataGrid.tsx

```diff
@@ -78,24 +80,26 @@ function DataGrid(props: DataGridProps) {
   // Add a column for the "string" row markers if needed
   useEffect(() => {
     setColumns(
-      isStringRowMarkers ? [{ id: "", title: "" }, ...columnsFromProps] : columnsFromProps,
+      isStringRowMarkers
+        ? [{ id: "", title: "", width: rowMarkersOptions.width }, ...columnsFromProps]
+        : columnsFromProps,
     );
-  }, [columnsFromProps, isStringRowMarkers]);
+  }, [columnsFromProps, isStringRowMarkers, rowMarkersOptions.width]);
```

Comment:
here about the useEffect, not sure about it but seems not necessary no ?

```ts
  const columns = useMemo(() => {
    if (isStringRowMarkers) {
      return [{ id: "", title: "", width: rowMarkersOptions.width }, ...columnsFromProps];
    }

    return columnsFromProps;
  }, [columnsFromProps, isStringRowMarkers, rowMarkersOptions.width]);
  ```

Reply:
`setColumns` is used for columns resizing